test/libsolidity/syntaxTests/metaTypes/codeAccessBase.sol

```diff
@@ -16,7 +16,4 @@ contract Test is Base {
     }
 }
 // ----
-// TypeError 4224: (165-188): Circular reference for contract code access.
-// TypeError 4224: (271-293): Circular reference for contract code access.
-// TypeError 4224: (381-404): Circular reference for contract code access.
-// TypeError 4224: (491-513): Circular reference for contract code access.
+// TypeError 7813: (165-188): Circular reference found.
```

Comment:
Well, `creationBase()` and `runtimeBase()` don't create an error anymore here. However, even when in different contracts, this results in only one error, even though both `Test` calls would cause one

Reply:
I am at this point not sure if that's okay

Reply:
> However, even when in different contracts, this results in only one error, even though both `Test` calls would cause one

I think this might actually be an example of https://github.com/ethereum/solidity/pull/10228#discussion_r587774557. I suspect that for both cycles the node added to `m_foundCycles` is `Base` and the second one overwrites the first.

Reply:
I'll investigate that

Reply:
I reversed the contract <-> referencee now but this contract still only gets one error, no matter in how many contracts it is split... I didn't investigate further yet.